paper.md

```diff
@@ -0,0 +1,178 @@
+
+# Notable features of HNN-core 
+
+HNN-core code enables the creation of a new and improved web-based GUI based on ipywidgets and voila that can be run remotely with port forwarding. HNN-core functionality also supports advanced simulations through scripting that are not currently possible in the GUI including:
+
+- The ability to record extracellular local field potentials from user defined positions, as well as voltages and synaptic currents from any compartment in the model
+- The ability to modify all features of the morphology and biophysical properties of any cell in the network
+- An API that enables complete control of cell-cell and drive-cell connectivity in the network
+- An API that allows for flexibility in defining the exogenous layer specific drive to the neocortical network
+- The ability to choose from multiple template models based on previous publications (`jones_2009_model()`{.python} [@jones2009quantitative], `law_2021_model()`{.python} [@law2022thalamocortical], `calcium_model()`{.python} [@kohl2022neural])
+- Built-in ERP optimization functionality designed for faster convergence 
+- The choice of two parallel backends for either parallelizing across cells to speed up individual simulations (MPI), or across trials to speed up batches of simulations (joblib)
+
+All of the code associated with HNN-core has been extensively documented at multiple levels,  including an API describing basic functions/parameters and examples of  use for hypothesis generation and/or testing. Specifically, we distribute tutorials that mimic the original GUI tutorial workflows for simulating ERPs and low frequency rhythms using HNN-core functions, with commentary on the known biophysical mechanisms of these signals. We also provide short and targeted “How to” examples that describe how to use specific functionality, such as plotting firing rates, or recording extracellular LFPs. 
+
+# Quick example code of running a simulation
+
+HNN-core has minimal dependencies which allows for effortless installation using the pip Python installer. In addition to numpy, scipy and matplotlib common in most libraries in the scientific Python stack, HNN-core uses Neuron for the cell and circuit modeling. Here, we demonstrate how the HNN-core interface can be used to quickly simulate and plot the net cortical dipole response to a brief exogenously evoked drive representing “feedforward” thalamocortical input. This input  (referred to as ‘evprox1’) effectively targets the proximal dendrites of the pyramidal neurons in L2/3 and L5, using the template neocortical model as in @jones2009quantitative.
```

Comment:
Same here, is there a standard for how python library names should be formatted?